roadmap.md

```diff
@@ -0,0 +1,14 @@
+# Roadmap
+
+**Deliverable**|**Status**|**Expected Delivery**|**Description**
```

Comment:
Can we sort the entries by Expected Delivery?

Reply:
+1

Reply:
will do